src/store/actions/actions-model.ts

```diff
@@ -1,17 +1,25 @@
 import { ActionIF } from '@/interfaces/store-interfaces/action-interface'
 import {
   AccountInformationIF, AddressIF, BusinessContactIF, CertifyIF, IncorporationAddressIF,
-  IncorporationAgreementIF, NameRequestIF, NameTranslationIF, OrgPersonIF, ResourceIF,
+  IncorporationAgreementIF, NameRequestIF, NameTranslationIF, OrgPersonIF, IncorporationResourceIF,
   ShareClassIF, CreateRulesIF, CreateMemorandumIF, ValidationDetailIF
 } from '@/interfaces'
 import { CoopType, CorpTypeCd } from '@/enums'
 
+export const setBusinessId: ActionIF = ({ commit }, businessId): void => {
+  commit('mutateBusinessId', businessId)
+}
+
+export const setFilingType: ActionIF = ({ commit }, filingType): void => {
+  commit('mutateFilingType', filingType)
+}
+
 export const setEntityType = ({ commit }, entityType: CorpTypeCd): void => {
   commit('mutateEntityType', entityType)
 }
 
-export const setCompanyResources = ({ commit }, companyResources: ResourceIF): void => {
-  commit('mutateCompanyResources', companyResources)
+export const setResources = ({ commit }, companyResources: IncorporationResourceIF): void => {
```

Comment:
Again, you probably need a union type here.

Reply:
100%, thanks!

Reply:
Still working through this one, same as above. Right now typescript is only accepting one and throwing errors on any resource getters that pulls on a value not in both interfaces.

Reply:
You might to explicitly cast props or return values. If you give me a specific example then I can try to figure it out. (I think I had to do something similar recently.)

Reply:
Here's what I had. Basically I couldn't directly do `item.comments = comments` due to the return value of fetchComments. My solution was the ternary on line 797.

![image](https://user-images.githubusercontent.com/10002889/135678889-a41a1ce2-9595-4c2a-b8a4-9b799855b9a7.png)

Reply:
Updated in next commit!